Trakt/Api/TraktApi.cs

```diff
@@ -1080,7 +1078,7 @@ private Task<Stream> PostToTrakt(string url, object data, TraktUser traktUser)
             CancellationToken cancellationToken,
             TraktUser traktUser)
         {
-            var jsonData = data == null ? string.Empty : _jsonSerializer.SerializeToString(data);
+            var jsonData = data == null ? string.Empty : JsonSerializer.Serialize(data, _jsonOptions);
```

Comment:
Possible improvement, serialize to stream or utf8 bytes, so we don't have the utf16 -> utf8 conversion

Reply:
Is this what you meant?

Reply:
Yes, but I think it's better to set the header here https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpcontent.headers?view=net-5.0#System_Net_Http_HttpContent_Headers
instead of modifying the default ones.

Just benched both and bytes may be the easier/faster solution
```cs
| Method |     Mean |    Error |    StdDev |   Median |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|------- |---------:|---------:|----------:|---------:|-------:|------:|------:|----------:|
|  Bytes | 433.3 ns |  8.07 ns |   8.29 ns | 434.0 ns | 0.1016 |     - |     - |     320 B |
| Stream | 743.4 ns | 55.06 ns | 161.47 ns | 639.6 ns | 0.1955 |     - |     - |     616 B |
```